src/Exceptionless.Core/Configuration/ElasticsearchOptions.cs

```diff
@@ -50,6 +53,10 @@ public class ConfigureElasticsearchOptions : IConfigureOptions<ElasticsearchOpti
             options.FieldsLimit = fieldsLimit > 0 ? fieldsLimit : 1500;
 
             options.EnableMapperSizePlugin = pairs.GetValueOrDefault("enable-size-plugin", _appOptions.Value.AppMode != AppMode.Development);
+
+            options.UserName = pairs.GetString("username", "");
+
+            options.Password = pairs.GetString("password", "");
```

Comment:
We shouldn't need the second parameter as it returns String.Empty if the key isn't set.